cluster/gce/config-default.sh

```diff
@@ -34,16 +34,15 @@ REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-true}
 PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 PREEMPTIBLE_MASTER=${PREEMPTIBLE_MASTER:-false}
 
-# TODO(#26183): Provide a way to differentiate master OS distro and node OS
-# distro.
-OS_DISTRIBUTION=${KUBE_OS_DISTRIBUTION:-gci}
+MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-gci}
+NODE_OS_DISTRIBUTION=${KUBE_NODE_OS_DISTRIBUTION:-debian}
 # By default a cluster will be started with the master on GCI and nodes on
 # containervm. If you are updating the containervm version, update this
 # variable.
 CVM_VERSION=container-v1-3-v20160604
 MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-}
 MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-google-containers}
-NODE_IMAGE=${KUBE_GCE_NODE_IMAGE:-"${MASTER_IMAGE}"}
+NODE_IMAGE=${KUBE_GCE_NODE_IMAGE:-"${CVM_VERSION}"}
 NODE_IMAGE_PROJECT=${KUBE_GCE_NODE_PROJECT:-"${MASTER_IMAGE_PROJECT}"}
```

Comment:
GCI and containervm happen to be sharing the image project, but ideally `NODE_IMAGE_PROJECT` should not rely on that assumption. So set it explicitly to `google-containers` too?